<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleDKSItem</class>
 <widget class="QFrame" name="ModuleDKSItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QPushButton" name="pushButton_FuncKey">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>7</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{min-height:24px;max-height:24px;};</string>
   </property>
   <property name="text">
    <string>未设置功能</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSwitch1">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../res.qrc">:/images/k/ic_add.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelSwitch2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../res.qrc">:/images/k/ic_add.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelSwitch3">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../res.qrc">:/images/k/ic_add.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelSwitch4">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../res.qrc">:/images/k/ic_add.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
